SECURITY.md

```diff
@@ -0,0 +1,12 @@
+# Security Policy
+
+## Supported Versions
+
+Security updates are applied only to the latest release.
```

Comment:
I think we should consider applying security updates to at least the previous major release.

Reply:
I've updated it to the latest two major versions. See if it is good or if it should be rephrased.